lib/Unix/PID/Tiny.pm

```diff
@@ -3,7 +3,7 @@ package Unix::PID::Tiny;
 use strict;
 use warnings;
 
-our $VERSION = '0.93';
+our $VERSION = '0.94';
```

Comment:
Can you bump this in the README and .pod file too? thanks!

Reply:
My bad! Sure. :)